### PR TITLE
UART

### PR DESCRIPTION
**Pull request by rusefillc**
UART is useful at least while we are trying to get HW to work

while this PR adds helpful flexibility I was unable to see UART actually working :(

### CODE REVIEW

File: firmware/boards/f1_dave/port.cpp [hunk 1]
```diff
@@ -64,3 +64,10 @@ void SetConfiguration(const Configuration& newConfig)
 {
     // TODO: implement me!
 }
+
+void StartIo() {
+    // stm32 TX - dongle RX often White
+    palSetPadMode(GPIOA, 9, PAL_MODE_STM32_ALTERNATE_PUSHPULL );
```
Comment (mck1117) [inline firmware/boards/f1_dave/port.cpp:70]: configure these in board.h, see f0 module:

image

**Comment by rusefillc** [comment]
@mck1117 board.h approach depends on board.c approach with stm32_gpio_init

I am not sure why but all F1x demos in ChibiOS are not using stm32_gpio_init which is part of the board.h mode mapping approach used in f0

**Comment by mck1117** [comment]
Right - the F1 has a different (maybe unique to F1?) GPIO peripheral, so the init options are somewhat different (worse) to those on F0/2/3/4/7 etc.

**Comment by rusefillc** [comment]
in light of what shall we merge this PR?

**Comment by rusefillc** [comment]
https://github.com/mck1117/wideband/commit/b910db45ba71f883af913f766250c797290e0721 is some depression manual bit magic in my opinion

**Pull request closed without merging**